ggml/src/ggml-sycl/softmax.cpp

```diff
@@ -133,15 +142,16 @@ static void soft_max_f32_submitter(const float * x, const float * mask, float *
         cgh.parallel_for(
             sycl::nd_range<3>(block_nums * block_dims, block_dims),
             [=](sycl::nd_item<3> item_ct1) [[intel::reqd_sub_group_size(WARP_SIZE)]] {
-                soft_max_f32<vals_smem, ncols_template, block_size_template>(x, mask, dst, ncols_par,
+                soft_max_f32<vals_smem, ncols_template, block_size_template, T>(x, mask, dst, ncols_par,
```

Comment:
Is it needed to explicitly mention the type T here and below? I expect it would be deduced thanks to the `mask` argument.